src/views/guidelines/guidelines.jsx

```diff
@@ -0,0 +1,51 @@
+var React = require('react');
+var FormattedMessage = require('react-intl').FormattedMessage;
+
+var render = require('../../lib/render.jsx');
+var Page = require('../../components/page/page.jsx');
+var Box = require('../../components/box/box.jsx');
+
+require('./guidelines.scss');
+
+var Guidelines = React.createClass({
+    type: 'Guidelines',
+    render: function () {
+        return (
+            <Box title={<FormattedMessage id='guidelines.title' />}>
+                <div className="bottom">
+                    <p><FormattedMessage id='guidelines.header' /></p>
+                    <ul>
+                        <li>
+                            <b><FormattedMessage id='guidelines.respectheader' /></b>
```

Comment:
@carljbowman thoughts on these being `h4` instead of `b`? that way it'd look like:
![screencapture-localhost-8333-community_guidelines-1461350018840](https://cloud.githubusercontent.com/assets/1815772/14751336/5859f6ca-0897-11e6-8e10-60b7ab3bc189.png)